packages/core/src/motify.tsx

```diff
@@ -9,14 +14,16 @@ export default function motify<
   Props extends ViewProps | TextProps | ImageProps,
   Style = Props['style'] extends StyleProp<infer S> ? S : Record<string, any>
 >(ComponentWithoutAnimation: ComponentType<Props>) {
-  const Component = Animated.createAnimatedComponent(ComponentWithoutAnimation)
+  const AnimatedComponent = Animated.createAnimatedComponent(
+    ComponentWithoutAnimation
+  )
 
   const withAnimations = () =>
     //  we might use these later
     // outerProps?: ExtraProps
     {
       const withStyles = forwardRef<
-        ComponentType<Props>,
+        Component<Props>,
```

Comment:
What I'm still not sure about here: does `Props` encapsulate the types for a React component's ref?

Reply:
I wonder if it should really be `typeof ComponentWithoutAnimation`.